public-ui/src/assets/images/all-of-us-logo.svg

```diff
@@ -0,0 +1,207 @@
+			s-9.9,4.6-9.9,10.4s4.1,10.4,9.9,10.4S389.2,202.7,389.2,196.9z M394.6,196.9c0,8.3-6.3,15.2-15.3,15.2s-15.2-6.8-15.2-15.1
+			c0-8.3,6.3-15.2,15.3-15.2S394.6,188.6,394.6,196.9z M329,204l7.2-21.8h4.3l7.2,21.8l7-21.7h5.4l-10.2,29.4h-4.4l-7.2-21l-7.2,21
+			h-4.4l-10.2-29.4h5.6L329,204z"/>
+	</defs>
+	<clipPath id="SVGID_2_">
+		<use xlink:href="#SVGID_1_"  style="overflow:visible;"/>
+	</clipPath>
+	<g class="st3">
+		<defs>
+			<rect id="SVGID_3_" x="99.8" y="78.5" width="591.2" height="591.2"/>
+		</defs>
+		<clipPath id="SVGID_4_">
+			<use xlink:href="#SVGID_3_"  style="overflow:visible;"/>
+		</clipPath>
+		<rect x="308.1" y="173.4" class="st4" width="307.8" height="46.9"/>
+	</g>
+</g>
+<g>
+	<defs>
+		<path id="SVGID_5_" d="M293.9,192.4L293.9,192.4c0-2.8-1.9-4.3-5.1-4.3h-6.4v8.4h6.5C292.1,196.5,293.9,194.8,293.9,192.4z
+			 M301.3,211.5h-7.5l-6.3-9.4h-5.1v9.4H276v-29.2h13.4c6.9,0,11,3.6,11,9.6v0.1c0,4.7-2.5,7.7-6.3,9.1L301.3,211.5z M270.3,211.5
+			H248v-29.2h22.1v5.7h-15.7v5.9h13.8v5.7h-13.8v6.1h15.9V211.5z M241,211.5h-6.4v-11.7h-11.9v11.7h-6.4v-29.2h6.4v11.6h11.9v-11.6
+			h6.4V211.5z M211.1,206.8c-3,3.2-6.3,5.2-12,5.2c-8.6,0-15-6.6-15-15v-0.1c0-8.3,6.3-15.1,15.2-15.1c5.5,0,8.8,1.8,11.5,4.5
+			l-4.1,4.7c-2.3-2-4.6-3.3-7.5-3.3c-4.9,0-8.5,4.1-8.5,9.1v0.1c0,5,3.5,9.2,8.5,9.2c3.3,0,5.4-1.3,7.7-3.4L211.1,206.8z
+			 M173.1,192.4L173.1,192.4c0-2.8-1.9-4.3-5.1-4.3h-6.4v8.4h6.5C171.3,196.5,173.1,194.8,173.1,192.4z M180.5,211.5H173l-6.3-9.4
+			h-5.1v9.4h-6.4v-29.2h13.4c6.9,0,11,3.6,11,9.6v0.1c0,4.7-2.5,7.7-6.3,9.1L180.5,211.5z M139.1,199.3l-3.9-9.5l-3.9,9.5H139.1z
+			 M150.7,211.5H144l-2.7-6.6H129l-2.7,6.6h-6.6l12.5-29.4h5.9L150.7,211.5z M116.5,211.5H94.3v-29.2h22.1v5.7h-15.7v5.9h13.8v5.7
+			h-13.8v6.1h15.9V211.5z M88.7,202.9c0,5.8-4.4,9.1-10.7,9.1c-4.4,0-8.9-1.5-12.4-4.7l3.8-4.6c2.6,2.2,5.4,3.6,8.7,3.6
+			c2.6,0,4.2-1,4.2-2.8v-0.1c0-1.6-1-2.5-5.9-3.7c-5.9-1.5-9.7-3.1-9.7-8.9v-0.1c0-5.3,4.3-8.8,10.2-8.8c4.3,0,7.9,1.3,10.9,3.7
+			l-3.3,4.8c-2.6-1.8-5.1-2.9-7.6-2.9c-2.5,0-3.8,1.1-3.8,2.5v0.1c0,1.9,1.3,2.5,6.3,3.8C85.3,195.6,88.7,197.7,88.7,202.9
+			L88.7,202.9z M61.8,211.5H39.6v-29.2h22.1v5.7H45.9v5.9h13.8v5.7H45.9v6.1h15.9V211.5z M27.2,192.4L27.2,192.4
+			c0-2.8-1.9-4.3-5.1-4.3h-6.4v8.4h6.5C25.4,196.5,27.2,194.8,27.2,192.4z M34.6,211.5h-7.5l-6.3-9.4h-5.1v9.4H9.4v-29.2h13.4
+			c6.9,0,11,3.6,11,9.6v0.1c0,4.7-2.5,7.7-6.3,9.1L34.6,211.5z"/>
+	</defs>
+	<clipPath id="SVGID_6_">
+		<use xlink:href="#SVGID_5_"  style="overflow:visible;"/>
+	</clipPath>
+	<g class="st5">
+		<defs>
+			<rect id="SVGID_7_" x="-209.2" y="115.6" width="591.2" height="591.2"/>
+		</defs>
+		<clipPath id="SVGID_8_">
+			<use xlink:href="#SVGID_7_"  style="overflow:visible;"/>
+		</clipPath>
+		<rect x="1" y="173.4" class="st6" width="308.6" height="46.9"/>
+	</g>
+</g>
+<g>
+	<line class="st7" x1="10.2" y1="163.9" x2="607.5" y2="163.9"/>
+</g>
+</svg>
```

Comment:
A lot of the assets/images included below (all?) seem unneeded for the data browser.

Reply:
My ui has a bunch of images they are not sure of so i am going to use these as placeholders